src/lib/isTaxID.js

```diff
@@ -1226,15 +1238,26 @@ export default function isTaxID(str, locale = 'en-US') {
     if (locale in sanitizeRegexes) {
       strcopy = strcopy.replace(sanitizeRegexes[locale], '');
     }
-    if (!taxIdFormat[locale].test(strcopy)) {
-      return false;
-    }
 
-    if (locale in taxIdCheck) {
-      return taxIdCheck[locale](strcopy);
+    // Check if locale has multiple TIN
+    if (locale in multipleTinLocale) {
+      if (!localeOption) throw new Error(`Locale requires localeOption ${localeOption}`);
+
+      if (!(multipleTinLocale[locale][localeOption])) throw new Error(`Locale Option ${localeOption} not available for specified locale ${locale}`);
+
+      if (!(taxIdFormat[locale][localeOption].test(strcopy))) return false;
+    } else {
+      if (localeOption || localeOption === '') throw new Error(`Locale ${locale} do not have localeOption '${localeOption}'`);
```

Comment:
```suggestion
      if (localeOption) throw new Error(`ocale ${locale} does not support localeOption '${localeOption}'`);
```

Reply:
Removing `localeOption === '' ` would result in returning false if the developer passes an empty string as a third argument, when locale is not in `multipleTinLocale`
Even it might be better if we restrict it to further to avoid passing null or undefined as third argument.

Reply:
That would be good, but we should do that earlier in the method (when we throw the error for `!localeOption`)